weave-js/src/components/PagePanelComponents/Home/Browse3/pages/ObjectVersionsPage.tsx

```diff
@@ -153,66 +175,138 @@ const ObjectVersionsTable: React.FC<{
   objectVersions: ObjectVersionSchema[];
   usingLatestFilter?: boolean;
 }> = props => {
+  // `showPropsAsColumns` probably needs to be a bit more robust
+  const showPropsAsColumns = !props.usingLatestFilter;
   const rows: GridRowsProp = useMemo(() => {
+    const vals = props.objectVersions.map(ov => ov.val);
+    const flat = prepareFlattenedDataForTable(vals);
+
     return props.objectVersions.map((ov, i) => {
+      let val = flat[i];
+      if (ov.baseObjectClass === DATASET_BASE_OBJECT_CLASS) {
+        // We don't want to show the rows column for datasets
+        // because it is redundant. Probably want a more generic
+        // solution here in the future. Maybe exclude table refs?
+        val = _.omit(val, 'rows');
+      }
+      // We don't want to show name (because it is the same as the object id)
+      val = _.omit(val, 'name');
```

Comment:
If we omit the name here, we can't display it when the user sets this intentionally. Moreover, I can't create an object in the table that produces an "id", so there is no risk of a collision between "name" and "id"... @tssweeney this is due to changes in the SDK (??) that make it now okay to remove this omit?